src/sentry/static/sentry/app/components/alert.jsx

```diff
@@ -59,10 +45,13 @@ const AlertWrapper = styled('div', {shouldForwardProp: isPropValid})`
   border: 1px solid ${p => p.theme.borderDark};
   align-items: center;
 
-  ${p =>
-    p.system
-      ? p.type && getSystemAlertColorStyles(p.theme.alert[p.type])
-      : p.type && getAlertColorStyles(p.theme.alert[p.type], p.theme.gray5)};
+  a {
+    color: ${p => p.theme.textColor};
+    border-bottom: 1px dotted ${p => p.theme.textColor};
+  }
+
+  ${p => getAlertColorStyles(p.theme.alert[p.type])} ${p =>
+      p.system && getSystemAlertColorStyles(p.theme.alert[p.type])};
```

Comment:
I don't really know how to get prettier to stop chewing this up 🤷‍♀️

Reply:
you can try adding a semicolon after maybe